e2e/cypress/integration/homepage.js

```diff
@@ -1,6 +1,87 @@
 describe('Ursus Homepage', () => {
+  beforeEach(() => { 
+    cy.visit('https://digital.library.ucla.edu/')
+  })
+
   it('Visits the Homepage', () => {
+    cy.percySnapshot();
+  });
+
+// Navbar
+
+  it('Logo', () => {
+    cy.get('.site-navbar__logo').click({ force: true });
+  });
+
+  it('About', () => {
+    cy.contains('a', 'About').click({ force: true });
+    cy.url().should('include', '/ursus_about');
+    cy.contains('h1', 'About Us');
+    cy.percySnapshot();
+  });
+
+// Footer / Static pages
+
+  it('Copyright and Collections', () => {
+    cy.contains('a', 'Copyright and Collection').click({ force: true });
+    cy.url().should('include', '/copyrights_and_collections');
+    cy.contains('h1', 'Copyrights and Collections');
+    cy.percySnapshot();
+  });
+
+  it('Privacy Policy', () => {
     cy.visit('/')
-    cy.percySnapshot()
-  })
-})
\ No newline at end of file
+    cy.contains('a', 'Privacy Policy').click({ force: true });
+    cy.url().should('include', '/privacy_policy');
+    cy.contains('h1', 'Privacy');
+    cy.percySnapshot();
+  });
+
+  it('Contact', () => {
+    cy.contains('a', 'Contact Us').click({ force: true });
+    cy.url().should('include', '/contact');
+    cy.contains('h1', 'Contact');
+    cy.percySnapshot();
+  });
+
+  it('Logo', () => {
+    cy.get('.site-footer__logo').click({ force: true });
+
+    cy.get('[data-cy=main-library-site]').then(function ($a) {
+      // extract the fully qualified href property
+      const href = $a.prop('href')
+      // make an http request for this resource outside of the browser
+      cy.request(href)
+      // drill into the response body
+      .its('body')
+      // and assert that its contents have the <html> response
+      .should('include', '<h3>Library Resources Quicklinks</h3>')
+      .and('include', '<h2 class="pane-title">Start your remote research</h2>')
+    });
+  });
+
+  it('Visit Our Legacy Site', () => {
+    cy.contains('a[href="http://digital2.library.ucla.edu/"]', 'Visit Our Legacy Site').click({ force: true });
```

Comment:
We don't want to actually visit digital2 during the test - just make sure the link is there.